examples/UI/bower_components/angular-resource/.bower.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Yes I'm pretty sure I can remove the bower_components that is used by CampaginBudgetsDev and replace all the dependencies with CDN, just have to keep in mind the versions I was using. Ill work on that today. I think for now this is a good approach, later down the line when we develop a bigger UI we can show our users how to convert the developer app into a distribution app. For that you need to have NPM installed to install Gulp with a bunch Gulp node_modules dependencies. Attached is a walk-through on how to do that. 
[GULP Instructions to make a Destribution Folder.pdf](https://github.com/venediktov/vanilla-rtb/files/1034122/GULP.Instructions.to.make.a.Destribution.Folder.pdf)